packages/core/chrome/core-chrome-browser-internal/src/doc_title/doc_title_service.ts

```diff
@@ -36,7 +53,9 @@ export class DocTitleService {
   }
 
   private applyTitle(title: string | string[]) {
-    this.document.title = this.render(title);
+    const rendered = this.render(title);
+    this.document!.title = this.render(title);
```

Comment:
nit: we can reuse the var we just created:
```suggestion
    this.document!.title = rendered;
```